### PR TITLE
Use html.escape instead of cgi.escape

### PR DESCRIPTION
**Pull request by mvdbeek**
`cgi.escape` has been removed in Python 3.8.

### CODE REVIEW

File: nosehtml/plugin.py [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 import errno
 import io
 import os
 import sys
 import traceback
+try:
+    from html import escape
+except ImportError:
+    # fallback for python 2
+    from cgi import escaoe
```
Comment (nsoranzo) [inline nosehtml/plugin.py]:
```suggestion
    from cgi import escape
```

File: nosehtml/plugin.py [hunk 2]
```diff
@@ -80,13 +84,13 @@ def print_test(self, status, test, error=None):
                 print(u"<div><span class='label'>Status:</span> %s</div>" % unicodify(status), file=f.file)
             if test.capturedOutput:
                 print(u"<div><span class='label'>Output:</span> <a href=\"javascript:toggle('capture_%d')\">...</a></div>" % f.counter, file=f.file)
-                print(u"<div id='capture_%d' style='display: none'><pre class='capture'>%s</pre></div>" % (f.counter, unicodify(cgi.escape(test.capturedOutput))), file=f.file)
+                print(u"<div id='capture_%d' style='display: none'><pre class='capture'>%s</pre></div>" % (f.counter, unicodify(escape(test.capturedOutput))), file=f.file)
```
Comment (nsoranzo) [inline nosehtml/plugin.py]:
```suggestion
                print(u"<div id='capture_%d' style='display: none'><pre class='capture'>%s</pre></div>" % (f.counter, unicodify(escape(test.capturedOutput, quote=True))), file=f.file)
```

File: nosehtml/plugin.py [hunk 3]
```diff
@@ -80,13 +84,13 @@ def print_test(self, status, test, error=None):
...
             if hasattr(test, 'capturedLogging') and test.capturedLogging:
                 print(u"<div><span class='label'>Log:</span> <a href=\"javascript:toggle('log_%d')\">...</a></div>" % f.counter, file=f.file)
-                print(u"<div id='log_%d' style='display: none'><pre class='log'>%s</pre></div>" % (f.counter, unicodify(cgi.escape("\n".join(test.capturedLogging)))), file=f.file)
+                print(u"<div id='log_%d' style='display: none'><pre class='log'>%s</pre></div>" % (f.counter, unicodify(escape("\n".join(test.capturedLogging)))), file=f.file)
```
Comment (nsoranzo) [inline nosehtml/plugin.py]:
```suggestion
                print(u"<div id='log_%d' style='display: none'><pre class='log'>%s</pre></div>" % (f.counter, unicodify(escape("\n".join(test.capturedLogging), quote=True))), file=f.file)
```

File: nosehtml/plugin.py [hunk 4]
```diff
@@ -80,13 +84,13 @@ def print_test(self, status, test, error=None):
...
             if error:
                 print(u"<div><span class='label'>Exception:</span> <a href=\"javascript:toggle('exception_%d')\">...</a></div>" % f.counter, file=f.file)
-                print(u"<div id='exception_%d' style='display: none'><pre class='exception'>%s</pre></div>" % (f.counter, unicodify(cgi.escape(error))), file=f.file)
+                print(u"<div id='exception_%d' style='display: none'><pre class='exception'>%s</pre></div>" % (f.counter, unicodify(escape(error))), file=f.file)
```
Comment (nsoranzo) [inline nosehtml/plugin.py]:
```suggestion
                print(u"<div id='exception_%d' style='display: none'><pre class='exception'>%s</pre></div>" % (f.counter, unicodify(escape(error, quote=True))), file=f.file)
```

**Review by nsoranzo — APPROVED** [review]

**Comment by nsoranzo** [comment]
@mvdbeek Should I tag 0.4.6 ?

**Comment by mvdbeek** [comment]
That would be great!

**Pull request merged**